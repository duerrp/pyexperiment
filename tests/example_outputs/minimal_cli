Error: Not enough arguments.
usage: minimal_cli [-h] [-c CONFIG] [-o key value] [-i]
                   [--verbosity {DEBUG,INFO,WARNING,ERROR,CRITICAL}]
                   [{hello,world,help,test,show_tests,show_config,save_config,show_state,show_commands,activate_autocompletion}]
                   [argument [argument ...]]
