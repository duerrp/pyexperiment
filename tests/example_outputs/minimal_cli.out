Error: Not enough arguments.
usage: minimal_cli [-h] [-c CONFIG] [-o key value] [-i]
                   [--verbosity {DEBUG,INFO,WARNING,ERROR,CRITICAL}] [-v]
                   [{hello,world,help,test,show_tests,show_config,save_config,show_state,show_commands}]
                   [argument [argument ...]]
